schema/spacewalk/common/data/rhnTaskoBunch.sql

```diff
@@ -117,4 +117,10 @@ INSERT INTO rhnTaskoBunch (id, name, description, org_bunch)
 INSERT INTO rhnTaskoBunch (id, name, description, org_bunch)
    VALUES (sequence_nextval('rhn_tasko_bunch_id_seq'), 'mgr-forward-registration-bunch', 'Forward registrations to SUSE Customer Center', null);
 
+INSERT INTO rhnTaskoBunch (id, name, description, org_bunch)
+VALUES (sequence_nextval('rhn_tasko_bunch_id_seq'), 'update-payg-data-bunch', 'Runs update-payg-data
+Parameters:
+- integer parameter payg instance ID
+- without parameter updates data for all instances', null);
```

Comment:
IIRC this text is user-visible. Is it intended or should this be a comment for developers only?

Reply:
AFAIK this text is only returned in one method of tasko xml rpc API.
The text that is shown in the UI is defined in another place [1].
I follow the same logic as other job definitions [2].

[1] e5f6a68#diff-44724356f81f311b199b24e5eea35f54fcefb6e7ea7ecf413f60b9823626c36aR23718-R23723
[2] https://github.com/uyuni-project/uyuni/blob/master/schema/spacewalk/common/data/rhnTaskoBunch.sql#L20-L31